app/models/course.rb

```diff
@@ -248,9 +248,8 @@ def new_draft_attributes
   scope :findable, -> { joins(:site_statuses).merge(SiteStatus.findable) }
   scope :with_vacancies, -> { joins(:site_statuses).merge(SiteStatus.with_vacancies) }
   scope :with_salary, lambda {
-    where(
-      program_type: %i[school_direct_salaried_training_programme pg_teaching_apprenticeship scitt_salaried_programme higher_education_salaried_programme]
-    )
+    program_types = Program.where_salaried
+    where(program_type: program_types)
```

Comment:
Small thing I would put everything under one line. `where(program_type: Program.where_salaried)` as this is easy to read.

Reply:
I feel the separate lines are easier to read - not fussed either way 🤷‍♂️